pkg/http/api.go

```diff
@@ -263,9 +302,14 @@ func GetSecretNames(host string, verifyTLS bool, apiKey string, project string,
 	params = append(params, queryParam{Key: "config", Value: config})
 	params = append(params, queryParam{Key: "include_dynamic_secrets", Value: strconv.FormatBool(includeDynamicSecrets)})
 
-	statusCode, _, response, err := GetRequest(host, verifyTLS, apiKeyHeader(apiKey), "/v3/configs/config/secrets/names", params)
+	url, err := generateURL(host, "/v3/configs/config/secrets/names", params)
 	if err != nil {
-		return nil, Error{Err: err, Message: "Unable to fetch secret names", Code: statusCode}
+		return nil, Error{Err: err, Message: "Unable to generate url"}
+	}
+
+	statusCode, _, response, err := GetRequest(url, verifyTLS, apiKeyHeader(apiKey))
+	if err != nil {
+		return nil, Error{Err: err, Message: "Unable to generate url"}
```

Comment:
nit: This error message probably shouldn't have changed

Reply:
I was fully confident that I messed up at least one of these. Now that this has been caught, I can relax!

Reply:
No worries - I went through 'em all too. That's what reviews are for 👍